### ISSUE TITLE
Mystical Agriculture Entities/Items missing. As well as some cyclic and stupidthings

### ISSUE THREAD

**Issue by cjbirlet1-catamount-wcu-edu** [adam9899/Issues#165]
Part of the mystical agriculture items are missing in the most recent version, such as emerald, wither skeleton essence, creeper essence, and constantan essence, as well as their seeds.

Furthermore, cyclic items: cyclic magic sprinkler and charm related things, as well as the uncrafting table are removed

Lastly, stupidthings acid is no longer there as well
![missing](https://user-images.githubusercontent.com/46974415/73478329-e3bf2680-4363-11ea-80de-9134f05fc2f8.png)

**Comment by cjbirlet1-catamount-wcu-edu** [comment]
For reference I deleted config, resource, and the mod folders that were used by the server for version 1.3 then downloaded the current 1.3.2 and dropped those in, in an attempt to avoid overfilling with duplicate mod jars, .00001 versions apart

**Comment by H4nSolo** [comment]
Its normal.....
after update to Version 1.3.2 many recept has changes...
look changelogs ;)

**Comment by ByteTemplar** [comment]
Indeed, next time, before updating, you really should read the changelog first! 
Ive seen other "issues" made about things, clearly noted in changelog in many different modpack issue trackers. Scary how few people these days read patch notes of things...

**Comment by cjbirlet1-catamount-wcu-edu** [comment]
> 
> 
> Indeed, next time, before updating, you really should read the changelog first!
> Ive seen other "issues" made about things, clearly noted in changelog in many different modpack issue trackers. Scary how few people these days read patch notes of things...

if you could, post the link to the changelog?

I could not find anything about it, furthermore i was a confused why the quests maintained the removed items.

Lastly, if they changed the emerald seeds to shop only, it wouldn't remove them, and break the quest, it would just make them unavailable, but all of those crops are now deleted...


Edit: I found the changelog, but the proceeding issues still stand

**Comment by ByteTemplar** [comment]
The line in changelog suggests, that the seeds will become available after progressing and unlocking them in Shop (they arent even visible in JEI). So you cant just buy em at the start of the game and get diamonds immediately...

    - nether star, gold, emerald, blaze and diamond seeds are no longer craftable but are available as shop/quest unlockables

_Edit:_
I made a test world and completed all the quests by using a command and the seeds did appear in the Shop! The items wont be available at all unless they get unlocked, which is why they got removed from the world in your game.

Working as intended...
I dont know what exact progression needs to be made to unlock them, that remains a mystery.

**Comment by cjbirlet1-catamount-wcu-edu** [comment]
> 
> 
> The line in changelog suggests, that the seeds will become available after progressing and unlocking them in Shop (they arent even visible in JEI). So you cant just buy em at the start of the game and get diamonds immediately...
> 
> ```
> - nether star, gold, emerald, blaze and diamond seeds are no longer craftable but are available as shop/quest unlockables
> ```
> 
> _Edit:_
> I made a test world and completed all the quests by using a command and the seeds did appear in the Shop! The items wont be available at all unless they get unlocked, which is why they got removed from the world in your game.
> 
> Working as intended...
> I dont know what exact progression needs to be made to unlock them, that remains a mystery.

I appreciate the help, somehow i have the diamond seeds progression, but not the emerald, i'll have to look into how to get to it
